### ISSUE TITLE
LPC is not robust under black-box attacks

### ISSUE THREAD

**Issue by jiachens** [KaidongLi/pytorch-LatticePointClassifier#1]
Dear authors,

We have personally contacted the first author Kaidong Li about this finding through email. However, we got no response. You tested the FGSM attack under the L-inf norm in the paper with an $\epsilon = 0.1$ and reported the results. The robust accuracy in the paper was surprisingly high, but the FGSM attack is a one-step attack that cannot fully utilize the attack budget. To confirm the results, we conducted two experiments.

We use your codebase to train LPC with the ResNet-50 backbone.
1.Transfer attack with PGD adversaries. 2. Nattack and SPSA attack. We use an $\epsilon = 0.05$, which is 1/2 of the original budget used in the paper. Both two experiments can degrade the robustness of LPC to around 45% accuracy. 

We also find that a uniformly distributed noise [-0.05,0.05] could degrade the robustness of LPC to around 65%, which is much lower than the reported robust accuracy.

We hope the authors could acknowledge our findings and we are happy to have any discussions.
